src/getCloseTag.ts

```diff
@@ -0,0 +1,27 @@
+export default function getCloseTag(text: string, excludedTags: string[]): string {
+    const regex = /<(\/?[a-zA-Z][a-zA-Z0-9:\-_.]*)(?:\s+[^<>]?[^\s/<>=]*)*\s?>/g;
```

Comment:
`<abc key=value>` would not work correctly.

Reply:
ah yes indeed it's broken, need to add many more tests, thanks for noticing
```js
// with my change:
'lol <a href="/ok"></a><abc key="value"> </'.match(/<(\/?[a-zA-Z][a-zA-Z0-9:\-_.]*)(?:\s+[^<>]*?[^\s/<>=]+?)*?\s?>/g)
(3) ["<a href="/ok">", "</a>", "<abc key="value">"]
// original regex
'lol <a href="/ok"></a><abc key="value"> </'.match(/<(\/?[a-zA-Z][a-zA-Z0-9:\-_.]*)(?:\s+[^<>]?[^\s/<>=]*)*\s?>/g)
["</a>"]
```